### DISCUSSION TITLE
FTL and Jump Redesign

### DISCUSSION THREAD

**Discussion by royfalk** [vegastrike/Vega-Strike-Engine-Source#886] [Ideas]
The issue
-------------
The current FTL (SPEC/Warp) and Jump design looks cool but there are some issues:

1. It makes little sense for a ship to be without FTL capability. Yeah, there are probably some maintenance drones for facilities, but even insystem stuff requires it.
2. It makes no sense for a ship to not come with the required SPEC capacitors for SPEC travel. We get thrusters when we buy a ship. Why not SPEC capacitors for it.
3. It makes no sense for a jump drive upgrade to not include additional SPEC capacitors (if needed) to make a jump.
4. In SPEC we don't use shields. Energy drain is really just the SPEC drive. Either the reactor is powerful enough or it's not. You can theoretically get away with a ftl_capacitor01 for a capital ship.

Suggestions
-----------------

**FTL**
1. Larger ships are slower in SPEC (acceleration, top speed). 
2. Bigger capacitors enable faster SPEC.
3. Damage to capacitor reduces acceleration, top speed.
4. Even when damaged, provide limp home capability at 10% C.
5. All ships come with a reasonable/play-worthy minimum SPEC as part of the drive.

**Jump Drive**
6. Larger ships are slower to jump (jump delay) same as WC. 
7. Bigger capacitors enable less delay.
8. Damage to capacitor introduces delay.
9. No limp home capability for jump drives - if your drive is destroyed, you're stranded in that system.
10. All ships come with a reasonable/play-worthy minimum capability as part of the drive. 

Note that some of the above will take some coding effort. The current engine doesn't support everything.

**Comment by BenjamenMeyer** [comment]
mostly agree

- jump drives should have the capability to run without a separate SPEC capacitor; SPEC capacitors should recharge the jump drive faster or enable more jumps
- if a jump drive runs out, then it you're stuck in that system; every system should have some method of recharging the jump if the SPEC capacitor/reactor cannot. You might have to limp to it though.

**Reply by royfalk** [reply]
>  jump drives should have the capability to run without a separate SPEC capacitor; SPEC capacitors should recharge the jump drive faster or enable more jumps

That was my idea but I see I didn't write it clearly enough.

> if a jump drive runs out, then it you're stuck in that system; every system should have some method of recharging the jump if the SPEC capacitor/reactor cannot. You might have to limp to it though.

So, this is really for VS style play. WC is a totally different beast. IIRC, if you have nowhere to refuel, you could theoretically find yourself stranded. However, I have not played it in nearly twenty years.

**Reply by BenjamenMeyer** [reply]
> > jump drives should have the capability to run without a separate SPEC capacitor; SPEC capacitors should recharge the jump drive faster or enable more jumps
> 
> That was my idea but I see I didn't write it clearly enough.

yes - sometimes it helps to re-iterate a concept. I do agree with you ;)
